datalake/src/main/java/com/sequenceiq/datalake/service/rotation/RedbeamsPollerRotationExecutor.java

```diff
@@ -42,6 +42,16 @@ public void finalize(PollerRotationContext rotationContext) {
         sdxRotationService.rotateRedbeamsSecret(rotationContext.getResourceCrn(), (RedbeamsSecretType) rotationContext.getSecretType(), FINALIZE);
     }
 
+    @Override
+    public void preValidate(PollerRotationContext rotationContext) throws Exception {
+
```

Comment:
I think it would be better to use flowEndpoint.getLastFlowByResourceCrn

Reply:
discussed, will leave it empty